app/app.tf

```diff
@@ -323,3 +322,14 @@ module "device" {
   docker_image        = var.device_image
   service_count       = var.device_service_desired_count
 }
+
+module "route53" {
```

Comment:
I like the interface for this module! makes opting into this easy if someone wants to go that route.